chpl/chpl-service/src/test/java/gov/healthit/chpl/changerequest/validation/WebsiteValidationTest.java

```diff
@@ -0,0 +1,83 @@
+package gov.healthit.chpl.changerequest.validation;
+
+import static org.junit.Assert.assertFalse;
+import static org.junit.Assert.assertTrue;
+
+import java.util.HashMap;
+import java.util.Map;
+
+import org.junit.Before;
+import org.junit.Test;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.mockito.Mockito;
+import org.mockito.MockitoAnnotations;
+
+import gov.healthit.chpl.changerequest.builders.ChangeRequestBuilder;
+import gov.healthit.chpl.changerequest.domain.ChangeRequest;
+import gov.healthit.chpl.permissions.ResourcePermissions;
+
+public class WebsiteValidationTest {
+
+    @Mock
+    private ResourcePermissions resourcePermissions;
+
+    @InjectMocks
+    private WebsiteValidation validator;
+
+    @Before
+    public void setup() {
+        MockitoAnnotations.initMocks(this);
+    }
+
+    @Test
+    public void isValid_NonDeveloperWebisiteIsIgnored() {
+        Mockito.when(resourcePermissions.isUserRoleDeveloperAdmin()).thenReturn(false);
+
+        assertTrue(validator.isValid(new ChangeRequestValidationContext(null, null)));
+    }
+
+    @Test
+    public void isValid_DeveloperNoDetails() {
+        Mockito.when(resourcePermissions.isUserRoleDeveloperAdmin()).thenReturn(true);
+
+        Map<String, Object> details = new HashMap<String, Object>();
+        ChangeRequest cr = new ChangeRequestBuilder()
+                .withId(1l)
+                .withDetails(details)
+                .build();
+
+        assertTrue(validator.isValid(new ChangeRequestValidationContext(cr, null)));
+    }
+
+    @Test
+    public void isValid_DeveloperWithDetailsValid() {
+        Mockito.when(resourcePermissions.isUserRoleDeveloperAdmin()).thenReturn(true);
+
+        Map<String, Object> details = new HashMap<String, Object>();
+        details.put("website", "http://www.test.com");
+
+        ChangeRequest cr = new ChangeRequestBuilder()
+                .withId(1l)
+                .withDetails(details)
+                .build();
+
+        assertTrue(validator.isValid(new ChangeRequestValidationContext(cr, null)));
+    }
+
+    @Test
+    public void isValid_DeveloperWithDetailsInvalid() {
+        Mockito.when(resourcePermissions.isUserRoleDeveloperAdmin()).thenReturn(true);
+
+        Map<String, Object> details = new HashMap<String, Object>();
```

Comment:
These tests appear to have duplicate code that could maybe be modularized. But, it is just tests, so not vital. Just a note.